package.json

```diff
@@ -56,7 +56,7 @@
     "eslint-config-brightspace": "^2.0.0",
     "eslint-plugin-unicorn": "^59",
     "glob-all": "^3",
-    "messageformat-validator": "^2",
+    "messageformat-validator": "^3.0.0-beta",
```

Comment:
So I guess we still need this to give us warnings about untranslated and missing terms? And is there a plan about when to move this to non-beta?

Reply:
Sounds good!